dir/path.go

```diff
@@ -1,64 +1,86 @@
 package dir
 
 import (
+	"errors"
+	"io/fs"
+
 	"github.com/opencontainers/go-digest"
 )
 
 const (
 	// SignatureExtension defines the extension of the signature files
 	SignatureExtension = ".sig"
+	// ConfigFile is the name of config file
+	ConfigFile = "config.json"
+	// LocalKeysDir is the directory name for local key store
+	LocalKeysDir = "localkeys"
+	// SigningKeysFile is the file name of signing key info
+	SigningKeysFile = "signingkeys.json"
+	// TrustPolicyFile is the file name of trust policy info
+	TrustPolicyFile = "trustpolicy.json"
+	// TrustStoreDir is the directory name of trust store
+	TrustStoreDir = "truststore"
 )
 
 // PathManager contains the union directory file system and methods
 // to access paths of notation
 type PathManager struct {
-	ConfigFS  UnionDirFS
-	CacheFS   UnionDirFS
-	LibexecFS UnionDirFS
+	ConfigFS     UnionDirFS
+	CacheFS      UnionDirFS
+	LibexecFS    UnionDirFS
+	UserConfigFS UnionDirFS
+}
+
+func errorHandler(path string, err error) string {
+	// if path does not exist, return path for creating file.
+	if err != nil && !errors.Is(err, fs.ErrNotExist) {
+		panic(err)
+	}
+	return path
 }
 
 // Config returns the path of config.json
-func (p *PathManager) Config() (string, error) {
-	return p.ConfigFS.GetPath("config.json")
+func (p *PathManager) Config() string {
+	return errorHandler(p.ConfigFS.GetPath(ConfigFile))
```

Comment:
To simplify the function. Because query path is used frequently, handle error all the time is a disaster. Also, when user opens the file of the path, the user needs to handle error at that moment.